zrml/prediction-markets/src/weights.rs

```diff
@@ -76,56 +76,34 @@ impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
     // Storage: unknown [0x3a7472616e73616374696f6e5f6c6576656c3a] (r:1 w:1)
     // Storage: MarketCommons Markets (r:1 w:1)
     // Storage: Balances Reserves (r:1 w:1)
-    // Storage: Tokens Accounts (r:1 w:0)
     // Storage: System Account (r:1 w:0)
     // Storage: MarketCommons MarketPool (r:1 w:0)
     // Storage: PredictionMarkets MarketIdsPerReportBlock (r:1 w:1)
-    // Storage: Tokens TotalIssuance (r:0 w:10)
     // Storage: PredictionMarkets Disputes (r:0 w:1)
-    fn admin_destroy_disputed_market(a: u32, b: u32, c: u32) -> Weight {
-        (0 as Weight)
-            // Standard Error: 221_000
-            .saturating_add((114_650_000 as Weight).saturating_mul(a as Weight))
-            // Standard Error: 221_000
-            .saturating_add((7_205_000 as Weight).saturating_mul(b as Weight))
-            // Standard Error: 290_000
-            .saturating_add((100_222_000 as Weight).saturating_mul(c as Weight))
-            .saturating_add(T::DbWeight::get().reads(7 as Weight))
-            .saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(a as Weight)))
-            .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(a as Weight)))
-            .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(b as Weight)))
-            .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(c as Weight)))
+    fn admin_destroy_disputed_market() -> Weight {
+        (161_945_000 as Weight)
+            .saturating_add(T::DbWeight::get().reads(6 as Weight))
+            .saturating_add(T::DbWeight::get().writes(5 as Weight))
```

Comment:
Wow, what a difference to before!

Reply:
Most likely this will change, as I have benchmaked them on a laptop which is not much powerful. once we are confident with reviews I can add benchmarks needed lable.

Reply:
I mean, it's now not dependent on benchmark components. That's great!